OneWelcomeExampleApp/ErrorMappings/ErrorMapper.swift

```diff
@@ -16,31 +16,31 @@
 class ErrorMapper {
     func mapError(_ error: Error, pinChallenge: PinChallenge? = nil, customInfo: CustomInfo? = nil) -> AppError {
         switch error.domain {
-        case ONGGenericErrorDomain:
+        case ErrorDomains.generic:
             return GenericErrorDomainMapping().mapError(error)
-        case ONGPinValidationErrorDomain:
+        case ErrorDomains.pinValidation:
             return PinValidationErrorDomainMapping().mapError(error)
-        case ONGAuthenticationErrorDomain:
-            if let pinChallenge = pinChallenge, error.code == ONGAuthenticationError.invalidPin.rawValue {
+        case ErrorDomains.pinAuthentication:
```

Comment:
this is wrong, should be: `ErrorDomains.authentication`

Reply:
Good catch!